charts/dvpe-deployment-gloo/templates/deployment.yaml

```diff
@@ -108,6 +108,16 @@ spec:
             timeoutSeconds: {{ .Values.deployment.spec.containers.readinessProbe.timeoutSeconds }}
             successThreshold: {{ .Values.deployment.spec.containers.readinessProbe.successThreshold }}
             failureThreshold: {{ .Values.deployment.spec.containers.readinessProbe.failureThreshold }}
+          livenessProbe:
+            httpGet:
+              path: {{ .Values.deployment.spec.containers.livenessProbe.httpGet.path }}
```

Comment:
There are no values defined for those properties in values.yaml

Please add corresponding properties and set pre-defined defaults where appropriate!

Reply:
I added the corresponding properties and default values in values.yaml.